### PR TITLE
updated manifest files based on requirement

### PR DESCRIPTION
**Pull request by harshad16**
updated manifest files based on the requirement
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: adviser/base/openshift-templates/dependency-monkey.yaml [hunk 1]
```diff
@@ -45,43 +42,34 @@ parameters:
     displayName: Beam width
     value: "1000000"
   - name: THOTH_DEPENDENCY_MONKEY_STACK_OUTPUT
-    required: false
```
Comment (pacospace) [inline adviser/base/openshift-templates/dependency-monkey.yaml:48]: why do we remove them?

Comment (harshad16) [reply adviser/base/openshift-templates/dependency-monkey.yaml:48]: argocd fails to deploy as it sees this variable as default if set to false. 
showed this issue during the demo as well.

Comment (pacospace) [reply adviser/base/openshift-templates/dependency-monkey.yaml:48]: You explained it today very well today indeed :) Thanks!! Yesterday I was still ignorant :D

**Pull request merged**